### PR TITLE
Add downloads badge

### PR DESCRIPTION
**Pull request by AdrianSosic**
We're gaining momentum, so why not show this in form of a badge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 [![Supports Python](https://img.shields.io/pypi/pyversions/baybe?style=flat-square&label=Supports%20Python&labelColor=96d7d2&color=ffdcb9)](https://pypi.org/project/baybe/)
 [![PyPI version](https://img.shields.io/pypi/v/baybe.svg?style=flat-square&label=PyPI%20Version&labelColor=96d7d2&color=ffdcb9)](https://pypi.org/project/baybe/)
+[![Downloads](https://img.shields.io/pypi/dm/baybe?style=flat-square&label=Downloads&labelColor=96d7d2&color=ffdcb9)](https://pypistats.org/packages/baybe)
```
Comment (AVHopp) [inline README.md:10]: Did you include this on your fork such that we can have a look?

File: README.md
Comment (AdrianSosic) [inline README.md:1]: @Scienfitz @AVHopp: you can see the rendered version on the Github main page by selecting the corresponding branch.

There is one more badge I'd like to add in the future: coverage. But for that, let's first push the numbers a little bit further up 🙃

File: README.md (hunk 1 above)
Comment (AdrianSosic) [reply README.md:10]: see [comment](https://github.com/emdgroup/baybe/pull/418#discussion_r1828890087)

**Review by AVHopp — APPROVED** [review]
Very cool :)

**Review by Scienfitz — APPROVED** [review]

**Pull request merged**